src/status_im/contexts/shell/jump_to/pr_error_indicator.cljs

```diff
@@ -0,0 +1,36 @@
+(ns status-im.contexts.shell.jump-to.pr-error-indicator
+  (:require [quo.core :as quo]
+            [react-native.reanimated :as reanimated]
+            [react-native.safe-area :as safe-area]
+            [utils.re-frame :as rf]))
+
```

Comment:
certainly feels like this code should live elsewhere. 
For example `src/status-im/debugging/pr-error-indicator`
and then require the encapsulated functionality into this namespace. It's unlikely someone will go into this specific view expecting to fine some sort of error logging mechanism like this.

Reply:
Breaking it down further might be nice too,
e.g 

`src/status-im/debugging/pr-error-indicator.errors`
```
(defn- chat-open-error
  []
  (let [current-chat-id (rf/sub [:chats/current-chat-id])
        view-id         (rf/sub [:view-id])]
    (when (and (not (#{:chat :profile} view-id)) current-chat-id)
      "Error: Chat is not closed. \n(messages will be marked read automatically).\nPlease create an issue with the steps that lead to this state.")))

(def ^:private const errors [chat-open-error])

(defn check-for-error 
 (first (filter (fn [e] (seq (e)) errors))) ;; not tested code but general idea is that we can easily add more checks to test for here.
```

`src/status-im/debugging/pr-error-indicator.view`
```
(:require [src/status-im/debugging/pr-error-indicator.errors :as errors])

(defn- f-view [error-message]
  (let [z-index       (reanimated/use-shared-value -1)]
    (if error-message
      (reanimated/animate-delay z-index 2 2000)
      (reanimated/animate-delay z-index -1 2000))
    [reanimated/touchable-opacity
     {:style    (reanimated/apply-animations-to-style
                 {:z-index z-index}
                 {:position           :absolute
                  :top                (safe-area/get-top)
                  :align-self         :center
                  :border-radius      5
                  :padding-horizontal 5
                  :background-color   :red})
      :on-press #(js/alert error-message)}
     [quo/text
      {:weight :regular
       :size   :paragraph-2
       :align  :center
       :style  {:color :white}} "Error"]]))

(defn view [] 
  (let [error-message (errors/check-for-error)] 
   [:f> f-view error-message]
  ))
```